x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/constants.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+const STEP_O1_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step01.description',
+  {
+    defaultMessage: 'Lowest precision and highest recall setting.',
+  }
+);
+
+const STEP_O2_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step02.description',
+  {
+    defaultMessage: 'Default. High recall, low precision.',
+  }
+);
+
+const STEP_03_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step03.description',
+  {
+    defaultMessage: 'Increasing phrase matching: half the terms.',
+  }
+);
+
+const STEP_04_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step04.description',
+  {
+    defaultMessage: 'Increasing phrase matching: three-quarters of the terms.',
+  }
+);
+
+const STEP_05_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step05.description',
+  {
+    defaultMessage: 'Increasing phrase matching requirements: all but one of the terms.',
+  }
+);
+
+const STEP_06_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step06.description',
+  {
+    defaultMessage: 'All terms must match.',
+  }
+);
+
+const STEP_07_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step07.description',
+  {
+    defaultMessage:
+      'The strictest phrase matching requirement: all terms must match, and in the same field.',
+  }
+);
+
+const STEP_08_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step08.description',
+  {
+    defaultMessage: 'Decreasing typo tolerance: advanced typo tolerance is disabled.',
+  }
+);
+
+const STEP_09_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step09.description',
+  {
+    defaultMessage: 'Decreasing term matching: prefixing is disabled.',
+  }
+);
+
+const STEP_10_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step10.description',
+  {
+    defaultMessage: 'Decreasing typo-tolerance: no compound-word correction.',
+  }
+);
+
+const STEP_11_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engine.relevanceTuning.precisionSlider.step11.description',
+  {
+    defaultMessage: 'Exact spelling matches only.',
+  }
+);
+
+export const STEP_DESCRIPTIONS = [
+  STEP_O1_DESCRIPTION,
+  STEP_O2_DESCRIPTION,
```

Comment:
These are using the letter "O" and not the number "0"

```suggestion
  STEP_01_DESCRIPTION,
  STEP_02_DESCRIPTION,
```

Reply:
fda;fklfdjafk;dls;adfjklsa;dlj

Reply:
Now that I've fixed this in dc3357 I'm going to throw myself into the sun

Reply:
it's too late, i'm already there 😎